### PR TITLE
Fixes to storage dependencies in CI

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_testcase.py [hunk 1]
```diff
@@ -82,13 +82,18 @@ def create_service_management(self, service_class):
...
+            account_key_arg = account_key or settings.STORAGE_ACCOUNT_KEY
+        except NameError:
+            account_key_arg = settings.STORAGE_ACCOUNT_KEY
         session = Session()
         service = service_class(
-            account_name,
-            account_key,
-            request_session=session,
+            "https://{}.blob.core.windows.net".format(account_name_arg),
+            credentials=ccount_key_arg
```
Comment (scbedd) [inline sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_testcase.py]: Should be `account` not `ccount`. Will push an update to this PR. Glad recordings work! :P

**Comment by chlowell** [comment]
Looks like autorest is incorrectly adding commits, e.g. in #5918

File: sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_testcase.py (hunk 1 above)
Comment (scbedd) [reply sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_testcase.py]: Fixed up.

**Pull request merged**